README.md

```diff
@@ -85,6 +86,8 @@ Property | Required | Description
 JAEGER_SERVICE_NAME | yes | The service name
 JAEGER_AGENT_HOST | no | The hostname for communicating with agent via UDP
 JAEGER_AGENT_PORT | no | The port for communicating with agent via UDP
+JAEGER_GRPC_TARGET | no | The target for communicating with collector via GRPC
```

Comment:
In gRPC it‘s called „target“. This can be any valid target at only host, host:port, dns://host:port or since c-ares is supported dns://remote-dns/host:port. Or even Unix pipe. See https://github.com/grpc/grpc/blob/master/doc/naming.md

Reply:
Added clarification. Hope it's fine for you.

Reply:
thx